packages/fuels/Cargo.toml

```diff
@@ -19,5 +19,17 @@ fuels-signers = { version = "0.17.0", path = "../fuels-signers" }
 fuels-test-helpers = { version = "0.17.0", path = "../fuels-test-helpers" }
 fuels-types = { version = "0.17.0", path = "../fuels-types" }
 
+[dev-dependencies]
+anyhow = "1.0.58"
+ctor = " 0.1"
+fuel-core = { version = "0.9", default-features = false }
+fuel-gql-client = { version = "0.9", default-features = false }
+fuels = { path = "../fuels", default-features = false }
```

Comment:
I don't think we need to import `fuels` here anymore, as this is inside `fuels/`.

```suggestion
```